content/tutorials/manage-the-rippled-server/installation/capacity-planning.md

```diff
@@ -1,117 +1,179 @@
 
 ## Network and Hardware
 
-Each `rippled` server in the XRP Ledger network performs all of the transaction processing work of the network. It is unknown when volumes will approach maximum network capacity. Therefore, the baseline hardware for production `rippled` servers should be similar to that used in Ripple's [performance testing](https://ripple.com/dev-blog/demonstrably-scalable-blockchain/).
+Each `rippled` server in the XRP Ledger network performs all of the transaction processing work of the network. Therefore, the baseline hardware for production `rippled` servers should be similar to that used in Ripple's [performance testing](https://ripple.com/dev-blog/demonstrably-scalable-blockchain/).
+
+Ensuring that your `rippled` server meets these network and hardware requirements helps achieve consistent, good performance across the XRP Ledger network.
+
 
 ### Recommendation
 
 For best performance in enterprise production environments, Ripple recommends running `rippled` on bare metal with the following characteristics:
 
 - Operating System: Ubuntu 16.04+
 - CPU: Intel Xeon 3+ GHz processor with 4 cores and hyperthreading enabled
-- Disk: SSD
+- Disk: SSD (7000+ writes/second, 10,000+ reads/second)
 - RAM:
 	- For testing: 8GB+
 	- For production: 32GB
 - Network: Enterprise data center network with a gigabit network interface on the host
 
-#### SSD Storage
+#### CPU Utilization and Virtualization
 
-SSD storage should support several thousand of both read and write IOPS. The maximum reads and writes per second that Ripple engineers have observed are over 10,000 reads per second (in heavily-used public server clusters), and over 7,000 writes per second (in dedicated performance testing).
+You'll get the best performance on bare metal, but virtual machines can perform nearly as well as long as the host hardware has high enough specs.
 
-#### CPU Utilization and Virtualization
-Ripple performance engineering has determined that bare metal servers achieve maximum throughput. However, it is likely that hypervisors cause minimal degradation in performance.
+#### Storage
 
-#### Network
+Here are some estimated `rippled` storage requirements:
 
-Any enterprise or carrier-class data center should have substantial network bandwidth to support running `rippled` servers. The minimum requirements are roughly 2Mbps transmit and 2Mbps receive for current transaction volumes. However, these can burst up to 100MBps transmissions when serving historical ledger and transaction reports. When a `rippled` server initially starts up, it can burst to over 20Mbps receive.
+- RocksDB stores around 8GB per day
+- NuDB stores around 12GB per day
```

Comment:
I didn't realize NuDB actually _stores_ more data than RocksDB. I guess it's a data compression thing?